sc2ts/info.py

```diff
@@ -1440,7 +1440,18 @@ class SampleGroupInfo:
     ts: tskit.TreeSequence
     attach_date: None
 
-    def draw_svg(self, size=(800, 600), time_scale=None, y_axis=True, mutation_labels=None, style=None, highlight_universal_mutations=None, x_regions=None, **kwargs):
+    def draw_svg(
+            self,
+            size=(800, 600),
+            time_scale=None,
+            y_axis=True,
+            mutation_labels=None,
+            style=None,
+            highlight_universal_mutations=None,
+            x_regions=None,
+            node_labels=None,
+            node_label_key=None,
```

Comment:
This is a nice idea, but it requires that the thing you want to add is in the metadata, and that the metadata is something you want to display "as is". How about we do something more direct like say
```
if node_labels == "strain":
    node_labels = {u: md["strain"] for ...
elif node_labels == "pango":
    ...
elif node_labels == "country":
   ...
```
That way we can tune it to make it easy to show the stuff we're using? This is just a debugging tool, so we can just add what we need quickly and not worry about long term stuff.

Reply:
Yep, that would work. I thought labelling by Pango + (abbreviated) country would be most useful, and we could easily have a "pango+country" option.

Reply:
sure

Reply:
Done. Should be able to merge.